cmd/nirvana/main.go

```diff
@@ -21,12 +21,14 @@ import (
 	"github.com/caicloud/nirvana/cmd/nirvana/client"
 	"github.com/caicloud/nirvana/cmd/nirvana/project"
 	"github.com/caicloud/nirvana/log"
+
 	"github.com/spf13/cobra"
 )
 
 var root = &cobra.Command{
-	Use:   "nirvana",
-	Short: "Nirvana toolchains",
+	Use:     "nirvana",
+	Short:   "Nirvana toolchains",
+	Version: "v0.3.0-alpha.0",
```

Comment:
IMHO I prefer automation than manual operation, the one who is responsible for making a release might forget to update the info here before he/she creates a new release.

In addition, `kubectl`, `kind`, `helm`(including but not limited to) these commands all provide a `version` subcommand, and they all inject the version information at build time:
* kubectl:  https://github.com/kubernetes/kubernetes/blob/8456a514f18925a4406e385f32b0a9fc659b7966/hack/lib/version.sh#L165-L178
* kind: https://github.com/kubernetes-sigs/kind/blob/86bcbff17fb2afff93f56479cccb97812970e67a/Makefile#L55
* helm: https://github.com/helm/helm/blob/82f739072cb3c94f26e3103e4524a1f74526342d/Makefile#L53-L56

I think this is a good convention to follow.

Reply:
But nirvana is just a simple and small project, you can simply install it by `go get` rather than downloading a binary file or executing another script, for other users who install it with `go get`, they can't even get the version information.

Similar tooling projects like Hugo are also hardcode the version information: https://github.com/gohugoio/hugo/blob/master/common/hugo/version_current.go